wolfcrypt/src/signature.c

```diff
@@ -54,31 +86,36 @@ int wc_SignatureGetSize(enum wc_SignatureType sig_type,
     (void)key_len;
 
     switch(sig_type) {
-#ifdef HAVE_ECC
         case WC_SIGNATURE_TYPE_ECC:
-        {
+#ifdef HAVE_ECC
             if (key_len >= sizeof(ecc_key)) {
```

Comment:
comparison against sizeof(ecc_key) is probably wrong.  ecc_key structure will differ in size for normal vs fastmath vs different bit sizes, FP_MAX_BITS etc.